src/core/Core.js

```diff
@@ -220,7 +221,7 @@ class Uppy {
     this.setState({meta: newMeta})
   }
 
-  updateMeta (data, fileID) {
+  setFileMeta (fileID, data) {
```

Comment:
should we add an error like this for one release?

```js
updateMeta (data, fileID) {
  throw new Error('The Uppy#updateMeta(data, fileID) method has been renamed to setFileMeta(fileID, data)')
}
```

Reply:
Yeah, I’ll do it, but I feel like before 1.0 and with a trivial API (not changing Tus naming or .upload), I think it’s fine to just announce in changelog and post. But it’s a good habit to develop, I agree.

Reply:
on second thought, with the amount of breaking changes in this pr it's probably not needed, since folks will have to inspect their code closely anyway

Reply:
Yeah, we’d also need to add a warning to every event ;-)